com.mudita.mudita-center.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: com.mudita.mudita-center
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: run.sh
+separate-locales: false
+rename-desktop-file: muditacenter.desktop
+rename-icon:
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=all
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.PowerManagement
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.gnome.SettingsDaemon
+  - --system-talk-name=org.freedesktop.login1
+modules:
+  - name: muditacenter
+    buildsystem: simple
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/mudita/mudita-center/releases/download/2.5.0/Mudita-Center.AppImage
+        sha256: 0fbef5c2020a8f9574d438741c5907f00a828c48927b48f8b12817881a2176a5
+      - type: file
+        path: run.sh
+      - type: file
+        path: muditacenter.desktop
```

Comment:
why is this not in the appimage?

Reply:
Thanks, I found one in there and will use that, I'd missed it initially.